tests/integration/test_insert_into_distributed/test.py

```diff
@@ -162,6 +174,15 @@ def test_inserts_local(started_cluster):
     assert instance.query("SELECT count(*) FROM local").strip() == '1'
 
 
+def test_inserts_single_replica(started_cluster):
+    node1.query(
+        "INSERT INTO distributed_one_replica VALUES ('2000-01-01', 1)",
+        settings={"insert_distributed_sync": "1", "prefer_localhost_replica": "0"},
```

Comment:
So it does not work with `prefer_localhost_replica=1`, kind of strange, but I guess OK